shell/platform/tizen/channels/platform_view_channel.cc

```diff
@@ -151,10 +156,28 @@ void PlatformViewChannel::HandleMethodCall(
         channel_->InvokeMethod("viewFocused", std::move(id));
       }
 
+      if (engine_ && engine_->text_input_channel) {
+        Ecore_IMF_Context* context =
+            engine_->text_input_channel->GetImfContext();
```

Comment:
Could you simply describe how you plan to handle this imf context event?
As you already know, this context already had handlers for input widget, not web-view.

Reply:
You are right. First of all, we have to check whether the webview is one of the flutter widgets. Then, we have to make the webivew to send the engine the events like setClient or clearClient. However, I have no idea how to solve this issue yet.

Reply:
All right. I don't have a specific idea either. Let's keep moving forward